plugin/v3/plugin.proto

```diff
@@ -41,6 +45,15 @@ message GetVersion {
   }
 }
 
+message GetSpecSchema {
+  message Request {}
+  message Response {
+    // Should be a valid JSON schema for the plugin own (nested) spec.
```

Comment:
```suggestion
    // Should be a valid JSON schema for the plugin's own (nested) spec.
```

Reply:
I cast thy `attributive noun` meaning for `plugin spec` used here!

Reply:
9e8d07bb634c781c024e7a461f3d963919fbadfc